api/server/server.go

```diff
@@ -740,6 +740,15 @@ func (s *Server) postImagesCreate(eng *engine.Engine, version version.Version, w
 		}
 	}
 
+	var (
+		opErr   error
+		useJSON = version.GreaterThan("1.0")
+	)
+
+	if useJSON {
```

Comment:
strictly no, but I saw that it was being duplicated lower in the function and putting this here reduces the number of if/else branches elsewhere. I can also make a `useJSON` variable so that we don't have to call `version.GreaterThan("1.0")` in 3 separate places.